userspace/libscap/scap.h

```diff
@@ -838,6 +838,11 @@ int32_t scap_get_stats(scap_t* handle, OUT scap_stats* stats);
 */
 int scap_get_modifies_state_ppm_sc(uint32_t * ppm_sc_array);
 
+/*!
+  \brief Take an array of `ppm_sc` as input and provide the associated array of events as output.
+*/
+int scap_get_events_from_ppm_sc(IN uint32_t* ppm_sc_array, OUT uint32_t* events_array);
```

Comment:
I'd specify the lengths of input and output parameters.

Reply:
Done :)